.travis.yml

```diff
@@ -52,6 +52,8 @@ cache:
     - $HOME/gcloud/
     - $HOME/site-library
 before_install:
+- REPORT_PATH=${TRAVIS_BRANCH}_$T{RAVIS_JOB_NUMBER};
```

Comment:
$T{RAVIS_JOB_NUMBER} -> ${TRAVIS_JOB_NUMBER}

Reply:
Also perhaps we should hide this text behind $TRAVIS_SECURE_ENV_VARS == true  so its not pointing to some non-existant bucket if someone external submits a PR.

Reply:
That's a a good idea.

Reply:
oh, and a good typo catch, this is why I should never make "minor changes" after checking that something is working....